### DISCUSSION TITLE
InplaceAdd  ---   It does not work with the filter or select Data by session

### DISCUSSION THREAD

**Discussion by ra-bedo11** [serenity-is/Serenity#6115] [Help] [answered]
![3](https://user-images.githubusercontent.com/29665710/143213139-2c967512-5640-4f3b-a5f0-72b3a5e0bf81.jpg)
![4](https://user-images.githubusercontent.com/29665710/143213303-64b939de-b2a7-4a4a-9d0b-059a2cdcbcfe.jpg)
![5](https://user-images.githubusercontent.com/29665710/143214855-d02b808b-055b-4453-beb4-eceb692c109f.jpg)

and This My Code

![1](https://user-images.githubusercontent.com/29665710/143214817-47a2d454-ef4c-4bd3-b189-9bb30d10b9e2.jpg)

![2](https://user-images.githubusercontent.com/29665710/143213522-47ed3b40-ae2c-4b57-be0e-228716fd7e3a.jpg)

Please : What is the solution Now?

**Comment by dfaruque** [comment, marked as answer]
you need to specify `[LookupEditor(typeof(aaaRow), InplaceAdd = true, DialogType = "YourDialogNameWithNameSpace")]`

**Reply by ra-bedo11** [reply]
Thank u  so much ,  its work

**Reply by edwardch** [reply]
Please could you mark this as the answer to help the rest of us with any questions :)

**Reply by axelcasauran** [reply]
@ra-bedo11 - have you make it work for "edit" feature after you select a value in the dropdown?
